### ISSUE TITLE
LK on Databricks / Katana: Cypher boxes were instable

### ISSUE THREAD

**Issue by mszel** [lynxkite/lynxkite#306]
For bigger graphs (I mean 200k nodes and 10m edges), a simple node count could not run in Cypher boxes. After filtering the graph, it was working fine (although very slow).

In some cases, it was running at the beginning but stopped running after a while (as the server is not on, I cannot send what was the exact error).

**Comment by mszel** [comment]
Also, uploading/reading csv files were a bit slow (after uploading and reading the csv, the workflow was thinking 3-4 minutes, and only after closing/reopening it, I could edit it further. It is not happening in pizzabox, there the import csv is working well.

**Comment by darabos** [comment]
Databricks and Katana integration are not part of the open-source repo. Could you please file these issues in the corresponding internal repos? Thanks!
